### DISCUSSION TITLE
Will other IaC solutions than Bicep like Pulumi be supported too?

### DISCUSSION THREAD

**Discussion by TechWatching** [Azure/azure-dev#25] [General]
>azd provision needs to know what to provision in Azure. The command looks for Bicep files in the infra folder.

azd seem to be made to work only with Bicep. Not everyone is using Bicep, some are using Pulumi, others are using Terraform. Is it planned for azd to support other IaC solutions in the future? I would love to have Pulumi supported.

**Comment by DamianFlynn** [comment]
This would be very important for many customers, terraform in specific is utilised a lot due to the vast number of providers supported

**Comment by markweitzel** [comment]
Yes. Terraform is on our roadmap. Likely we will get to this August timeframe.*
<sub>*That's an estimate. Tax, tag, and title not included. Professional driver on closed course....</sub>

**Reply by TechWatching** [reply]
What about Pulumi?

**Comment by savannahostrowski** [comment]
We've got an issue tracking Pulumi support (it's our most upvoted issue right now) https://github.com/Azure/azure-dev/issues/93. We will likely re-evaluate towards the end of this year.
